guide/process/steps/task.md

```diff
@@ -12,34 +13,34 @@ By default, the task's inputs are the previous step's outputs. Can be customized
     fullType: '&quot;task&quot;'
   }, {
     name: 'key',
-    description: '(optional) Key to identify this step.',
+    description: '(optional) Key to identify this step',
     fullType: 'string'
   }, {
     name: 'instanceHash',
-    description: 'Hash of the service\'s instance.',
+    description: 'Hash of the service\'s instance',
     fullType: 'string'
   }, {
     name: 'instance',
-    description: 'Information about the instance to run. (Ignored if instanceHash is present)',
+    description: 'Information about the instance to run. (Ignored if `instanceHash` is set)',
```

Comment:
Did you check that the ` is interpreted correctly, this will not display a code tag.
Should probably be something like
```suggestion
    description: 'Information about the instance to run. (Ignored if &lt;code&gt;instanceHash&lt;code&gt; is set)',
```
Same for the ones below